plugins/woocommerce-admin/client/products/sections/attributes-section.tsx

```diff
@@ -42,7 +45,12 @@ export const AttributesSection: React.FC = () => {
 				</>
 			}
 		>
-			<AttributeField { ...getInputProps( 'attributes' ) } />
+			{ /* eslint-disable-next-line @typescript-eslint/ban-ts-comment */ }
+			{ /* @ts-ignore */ }
+			<AttributeField
+				{ ...getInputProps( 'attributes' ) }
+				productId={ productId }
```

Comment:
This may not be the most ideal method for passing the `productId`. I believe I saw some discussion in Slack about this, so I'll likely update it next week.